admin/class-acc_user_importer-admin.php

```diff
@@ -154,14 +154,19 @@ class acc_user_importer_Admin {
 		'1906' => ['section' => 'NEWFOUNDLAND & LABRADOR', 'type' => 'child'],
 	);
 
-	//FIXME only the first 5 APIs have been created, the rest are bogus numbers
+	//TODO Intentially omitted FQME, which maps to 10.
 	private $sectionApiId = array (
 			'SQUAMISH' => '1',
 			'CALGARY' => '2',
 			'MONTRÉAL' => '3',
 			'OUTAOUAIS' => '4',
 			'OTTAWA' => '5',
-			'VANCOUVER' => '6');
+			'VANCOUVER' => '6'
```

Comment:
A comma is required at the end of each line, otherwise I believe you would get a parsing error.